src/legacy/core_plugins/ui_metric/public/services/telemetry_analytics.ts

```diff
@@ -39,28 +41,36 @@ export const createUiStatsReporter = (appName: string) => (
   }
 };
 
+export const trackUserAgent = (appName: string) => {
+  if (telemetryReporter) {
+    return telemetryReporter.reportUserAgent(appName);
```

Comment:
This popped up from someone's dev environment:

![image](https://user-images.githubusercontent.com/908371/68801659-c064c500-0619-11ea-80fb-7a9704143e44.png)

Reply:
As a note, this was caused by someone not running bootstrap